source/making_payments/index.html.md.erb

```diff
@@ -78,6 +78,11 @@ The format is variable. If you have an existing format, you can keep using it if
 
 The description must be no longer than 255 characters, and must not contain URLs.
 
+The `description` does not appear on your users' bank statements. To change your users' bank statements, you can either:
```

Comment:
I'm not sure this quite flows here - since the rest of the page is so much about the API. I think we can move it into the reporting /reconciliation bit...?

Reply:
Makes sense, thanks - I've moved it.